data/data.go

```diff
@@ -0,0 +1,72 @@
+package data
+
+// FilterType informs the name of the search type displayed on the website, the query retrieved from renderer and all the subtypes to pass to the logic
+type FilterType struct {
+	Name      string `bson:"name" json:"name"`
+	QueryType string `bson:"query_type" json:"query_type"`
+	SubTypes  string `bson:"sub_types" json:"sub_types"`
```

Comment:
Consider making `SubTypes` a `[]string` instead of `string`. If there is a specific reason this has to be a string on the struct consider adding a method to the `FilterType` struct to get the subtype values are a `[]string`.

Reply:
This has been done as mentioned in the previous comment :+1: